gapic/src/main/com/google/gapid/util/Colors.java

```diff
@@ -42,6 +44,18 @@ public static RGB rgb(double r, double g, double b) {
     return new RGB((int)(r * 255), (int)(g * 255), (int)(b * 255));
   }
 
+  public static RGB getRandomColor(int index) {
+    float hue = (float)Math.toDegrees(index * GOLDEN_ANGLE) % 360;
+    return new RGB(hue, 1f, 1f);
```

Comment:
Change the HSV params to `(hue, 0.8, 0.8)` and they're much closer. Maybe I'm now just mentally biased now, but I still prefer the original palette. However, I'd probably fail a blind test. :wink: